### DISCUSSION TITLE
How is zero handled for `enum Number`?

### DISCUSSION THREAD

**Discussion by thomas-k-cameron** [smithy-lang/smithy-rs#2195] [Q&A] [answered]
Current implementation allows zero to be either `Number::PosInt(0)` or `Number::NegInt(0)`.

Is this intended?

RFC30 converts 0 to `PosInt(0)`.
I thought that we need to clarify.

**Comment by Velfi** [comment, marked as answer]
I don't think it has an effect either way. Choose the one that feels right and don't worry about it imo.
